src/main/java/core/basesyntax/FiguresProducer.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FiguresProducer {
+    static final int NUMBER_OF_FIGURES = 4;//total numbers of different elements (Figures)
+    static final int POSSIBLE_ARGUMENT = 20;//max argument (excluded)
+    static final int POSSIBLE_FIGURES = 20;//max Figures can be generated(excluded)
```

Comment:
the name of the constant should display its purpose, not comment

Reply:
done